src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -176,31 +176,43 @@ public static void UpdateBackground(this ContentViewGroup platformView, IBorderS
 		// TODO: NET8 make this public for NET8.0
 		internal static void UpdateBackground(this EditText platformView, IView view)
 		{
-			var paint = view.Background;
-
-			if (paint.IsNullOrEmpty())
-			{
-				return;
-			}
-
 			// Remove previous background gradient if any
 			if (platformView.Background is MauiDrawable mauiDrawable)
 			{
 				platformView.Background = null;
 				mauiDrawable.Dispose();
 			}
 
-			// Android will reset the padding when setting a Background drawable
+			if (platformView.Background is LayerDrawable layerDrawable)
+			{
+				platformView.Background = null;
+				layerDrawable.Dispose();
+			}
+
+			// Android will reset the padding when setting a Background drawable	
 			// So we need to reapply the padding after
 			var padLeft = platformView.PaddingLeft;
 			var padTop = platformView.PaddingTop;
 			var padRight = platformView.PaddingRight;
 			var padBottom = platformView.PaddingBottom;
 
-			var previousDrawable = platformView.Background;
-			var backgroundDrawable = paint!.ToDrawable(platformView.Context);
-			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, previousDrawable! });
-			platformView.Background = layer;
+			var paint = view.Background;
+
+			Drawable? defaultBackgroundDrawable = null;
+
+			if (platformView.Context != null)
+				defaultBackgroundDrawable = ContextCompat.GetDrawable(platformView.Context, Resource.Drawable.abc_edit_text_material);
+
+			var previousDrawable = defaultBackgroundDrawable ?? platformView.Background;
```

Comment:
I think the case where `defaultBackgroundDrawable` can be null is if the Context of the native control is null. In case the PlatformView or the Context is null, we should not update anything.

Reply:
If that's the case, should we just exit this method early if `platformView.Context` is null? 

I think it'd be really weird/bad if we got here, and the Context was null. 

Maybe we just check for null on the platformView at the top of this method? and then write the rest of the code assuming it's not null?

Reply:
Added the Context validation at the top of the method.